SimG4Core/SensitiveDetector/src/SensitiveDetector.cc

```diff
@@ -16,7 +16,7 @@
 #include <sstream>
 
 SensitiveDetector::SensitiveDetector(const std::string& iname,
-                                     const DDCompactView& cpv,
+                                     const edm::EventSetup& es,
```

Comment:
@ianna , in this 'base' class we may add a method to access DDFilteredView. In that case, similar lines of code inside calorimeter and timing SD will be replaced by one line call to this method.

Reply:
@civanch - edm::EventSetup is common between before and after DD. The same ES record can be produced with either DD or DD4hep. DDFilteredView before is different from cms::DDFilteredView after.

Reply:
@ianna , I mean, that additional lines of code in SD classes, which uses DDFilteredView, all looks the same or very similar, so all this staff may be concentrated in one place - we may concentrate in this class SensitiveDetector the switch from old to new.